### DISCUSSION TITLE
Keybindings not working

### DISCUSSION THREAD

**Discussion by DrChilla** [mrkai77/Loop#606] [Q&A]
Hi, so i just downloaded Loop a few days ago, i really have liked it so far, I restarted my computer for the first time since though and now the keybindings don't work. the trigger key pulls up the square but then when I push any of the secondary keys it doesn't do anything it works with the mouse but I prefer the keys, I've tried to switch bindings and no luck, any thoughts?

Update: So I realized it is not working with a certain app (complete anatomy by 3dmedical) open, even if it's not the active window. I quit the app and it started working again. I reopened it and although it doesn't quite work perfectly with the app it at least tries now. I think perhaps it broke when I tried to change the app while it was in full screen and didn't work again until I got rid of the problem it had. I figured I would leave this here in case it helped someone else or helped with debugging.

**Comment by mrkai77** [comment]
Closing since it seems you solved this! Thanks for the information too :)
